samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Preparation;
+
+    /// # Summary
+    /// Example of a Repeat-until-success program implementing a circuit that achieves
+    /// exp(i⋅ArcTan(2)⋅Z) by Paetznick & Svore. Gate exp(i⋅ArcTan(2)⋅Z) is also know as
+    /// V gate.
+    /// # References
+    /// - [ *Adam Paetznick, Krysta M. Svore*,
+    ///     Quantum Information & Computation 14(15 & 16): 1277-1301 (2014)
+    ///   ](https://arxiv.org/abs/1311.1074)
+    /// For circuit diagram, see file RUS.png (to be added to README).
+    ///
+    /// The program executes a circuit on a "target" qubit using an "auxiliary" and 
+    /// "resource" qubit. The circuit consists of two parts (red and blue in image).
+    /// The goal is to measure Zero for both the auxiliary and resource qubit.
+    /// If this succeeds, the program will have effectively applied an 
+    /// Rz(arctan(2)) gate (also known as V_3 gate) on the target qubit.
+    /// If this fails, the program reruns the circuit up to <limit> times.
+    @EntryPoint()
+    operation ApplyRzArcTan2(
+        inputValue : Bool,
+        inputBasis: Pauli,
+        limit: Int
+    ) : (Bool, Result, Int) {
+        using ((auxiliary, resource, target) = (Qubit(), Qubit(), Qubit())) {
+            /// Prepare auxiliary and resource qubits in |+> state
```

Comment:
```suggestion
            // Prepare auxiliary and resource qubits in |+〉state
```

I am suggesting to use `//` comments for simple line comments and use `///` for potentially parsable documentation comments.

I am also suggesting the use of unicode in the sample comments to get closer to mathematical typesetting.